presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaPlugin.java

```diff
@@ -25,6 +25,7 @@
 {
     public static final Module DEFAULT_EXTENSION = binder -> {
         binder.install(new KafkaConsumerModule());
+        binder.install(new KafkaProducerModule());
```

Comment:
`PlainTextKafkaProducerFactory` is not extendable, there is no need to install this module here. Install it in `KafkaConnectorFactory` in a normal way.